pkg/dockerregistry/server/repositorymiddleware_test.go

```diff
@@ -9,8 +9,6 @@ import (
 	"testing"
```

Comment:
Please rename to `repository_test.go`.

Reply:
We already have `repository_test.go`.

Reply:
The `repository.go` [has been recently renamed](https://github.com/openshift/origin/commit/dfbbed95f6beed6173cc657742bdc55c77c34ab1#diff-df06600c3d2e39d32d9dcd471163e52e) from `repositorymiddlware.go`. As a consequence, `repositorymiddleware_test.go` is a leftover now lacking corresponding source file (and my editor's command `:A<CR>` to switch between test and non-test file does not work :cry:).

Can both `repository{middleware,}_test.go` be merged together?

Reply:
Well... Everybody would hate me: `repository_test.go` -> `registry_test.go`, `repositorymiddleware_test.go` -> `repository_test.go`.

Reply:
Not me :-) @legionus would you? Anybody else raise your hand!

Reply:
@dmage @miminar Nop. I'm fine ))